Makefile

```diff
@@ -42,6 +42,10 @@ codecheck:
 	flake8 grml2usb
 	isort --check-only grml2usb
 	black --check grml2usb
+	vulture grml2iso grml2usb test/grml2usb_test.py
+
+test:
+	pytest
```

Comment:
Won't that break Debian package builds? I think `dh` will try to run `make test` automatically, and `pytest` is not listed in `Build-Depends`?

Reply:
Good point, actually we're running `make codecheck` by default, so we need to add `vulture` to Build-Depends.
(Also vulture can't be executed on grml2iso, since that's a shell script and not python :))

I tried to tackle this in https://github.com/grml/grml2usb/pull/47